python/src/deltachat/account.py

```diff
@@ -190,36 +191,36 @@ def set_avatar(self, img_path):
             assert os.path.exists(img_path), img_path
             self.set_config("selfavatar", img_path)
 
-    def check_is_configured(self):
+    def check_is_configured(self) -> None:
         """ Raise ValueError if this account is not configured. """
         if not self.is_configured():
             raise ValueError("need to configure first")
 
-    def get_latest_backupfile(self, backupdir):
+    def get_latest_backupfile(self, backupdir) -> Optional[str]:
```

Comment:
```suggestion
    def get_latest_backupfile(self, backupdir: str) -> Optional[str]:
```

Reply:
Don't suggest additional annotations to this PR please, it's intentionally incomplete. I have left out places where I am not sure what types are allowed.

Reply:
sorry, I read this too late :(